tests/mgbench/benchmark.py

```diff
@@ -58,71 +205,281 @@ def filter_benchmarks(generators, patterns):
     patterns = copy.deepcopy(patterns)
     for i in range(len(patterns)):
         pattern = patterns[i].split("/")
-        if len(pattern) > 4 or len(pattern) == 0:
+        if len(pattern) > 5 or len(pattern) == 0:
             raise Exception("Invalid benchmark description '" + pattern + "'!")
         pattern.extend(["", "*", "*"][len(pattern) - 1 :])
         patterns[i] = pattern
     filtered = []
     for dataset in sorted(generators.keys()):
-        generator, tests = generators[dataset]
+        generator, queries = generators[dataset]
         for variant in generator.VARIANTS:
             is_default_variant = variant == generator.DEFAULT_VARIANT
             current = collections.defaultdict(list)
-            for group in tests:
-                for test_name, test_func in tests[group]:
-                    if match_patterns(dataset, variant, group, test_name, is_default_variant, patterns):
-                        current[group].append((test_name, test_func))
-            if len(current) > 0:
-                filtered.append((generator(variant), dict(current)))
+            for group in queries:
+                for query_name, query_func in queries[group]:
+                    if match_patterns(
+                        dataset,
+                        variant,
+                        group,
+                        query_name,
+                        is_default_variant,
+                        patterns,
+                    ):
+                        current[group].append((query_name, query_func))
+            if len(current) == 0:
+                continue
+
+            # Ignore benchgraph "basic" queries in standard CI/CD run
+            for pattern in patterns:
+                res = pattern.count("*")
+                key = "basic"
+                if res >= 2 and key in current.keys():
+                    current.pop(key)
```

Comment:
@gitbuda think a bit about this and make some proposal -> graphbench has to be updated